### DISCUSSION TITLE
Cannot locate svelte ...

### DISCUSSION THREAD

**Discussion by kavodlimited** [pdfslick/pdfslick#59] [Q&A]
npm install @pdfslick/svelte
npm error code E404
npm error 404 Not Found - GET https://registry.npmjs.org/@pdfslick%2fsvelte - Not found
npm error 404
npm error 404  '@pdfslick/svelte@*' is not in this registry.
npm error 404
npm error 404 Note that you can also install from a
npm error 404 tarball, folder, http url, or git url.
npm error A complete log of this run can be found in:

**Comment by van100j** [comment]
Please read the docs for more details on [using PDFSlick with Svelte](https://pdfslick.dev/docs/svelte).
